src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -97,7 +97,7 @@
                     <attribute name="summary">Select shipping methods for particular shipment</attribute>
                 </attribute>
                 <attribute name="method">PATCH</attribute>
-                <attribute name="path">/orders/{id}/select-shipping-methods</attribute>
+                <attribute name="path">/orders/{id}/shipments/{shipmentId}</attribute>
```

Comment:
We may be forced to move this endpoint to Shipment.xml class. There we can perhaps define 
```suggestion
                <attribute name="path">/orders/{token}/shipments/{id}</attribute>
```
and execute the command based mostly on `id` related to the shipment

Reply:
I think that changing `id` name will be not easy: https://github.com/api-platform/core/pull/2126